src/lightning/fabric/strategies/deepspeed.py

```diff
@@ -422,14 +422,18 @@ def save_checkpoint(
         engine.save_checkpoint(path, client_state=state, tag="checkpoint")
 
     def load_checkpoint(
-        self, path: _PATH, state: Optional[Dict[str, Union[Module, Optimizer, Any]]] = None
+        self,
+        path: _PATH,
+        state: Optional[Dict[str, Union[Module, Optimizer, Any]]] = None,
+        strict: Optional[bool] = True,
```

Comment:
I wonder if we shouldn't allow `**kwargs` here in general so that we don't need to add every argument manually?

Reply:
I think that we've seen some struggles in the past with mypy and inheritance with the kwargs pattern, not sure? In any case, the current code style is to write the kwargs out explicitly so I prefer to keep this consistency.